site/_posts/2013-11-20-inspectxml-dump-objects-as-xml.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: inspectXML – Dump objects as XML
 date: 2013-11-20
-tags: external operations cloudforms, vmware, rhev, kvm, cluster, vm, datastore, cf2, cf3, cfme, XML, XML format
```

Comment:
This one can't been seen in the diff, but there are embedded special characters in this line:
![__dev_manageiq_org_site__posts_2013-11-20-inspectxml-dump-objects-as-xml_md_—_manageiq_org](https://user-images.githubusercontent.com/52120/180078209-f8f8f37c-3ea2-41be-a06d-3fec66b91f67.png)